modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/BaseCKEditorConfigContributor.java

```diff
@@ -60,7 +60,8 @@ public void populateConfigJSONObject(
 				HtmlUtil.escape(
 					PortalUtil.getStaticResourceURL(
 						themeDisplay.getRequest(),
-						"/o/frontend-css-web/main.css")))
+						"/o/frontend-editor-ckeditor-web/ckeditor/skins" +
```

Comment:
Is this really needed? I'd expect CKEditor to load this automatically when passing in a different skin?

Reply:
Checked, we don't need this :D

Reply:
Did we set the skin to be the default in our build process in `liferay-ckeditor`?

Reply:
Yep https://github.com/liferay/liferay-ckeditor/blob/master/support/build-config.js#L9